src/com/trilead/ssh2/crypto/digest/HashForSSH2Types.java

```diff
@@ -1,35 +1,27 @@
-
 package com.trilead.ssh2.crypto.digest;
 
 import java.math.BigInteger;
+import java.security.DigestException;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
 
 /**
  * HashForSSH2Types.
- * 
+ *
  * @author Christian Plattner, plattner@trilead.com
  * @version $Id: HashForSSH2Types.java,v 1.1 2007/10/15 12:49:57 cplattne Exp $
  */
 public class HashForSSH2Types
 {
-	Digest md;
```

Comment:
Since this isn't a private field, we can't just delete it: there may be people out there who have some 'interesting' code that access it. I'd deprecate this class and create an updated clone of it with your changes in it and any un-needed fields removed.